spec/requests/api/v1/games_request_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+ RSpec.describe 'Games API' do
+   it "can send the game with the highest score" do
+     game = Game.create!(user_id: 7, score: 100, city: 'Chicago', state: 'Illinois', country: 'United States')
+
+     get "/api/v1/games/#{game.id}", params: {user_id: 7}
```

Comment:
We'll change this to '/api/v1/games/:user_id' for now and look into renaming controllers to be more restful.